fr/core-libraries/collections.rst

```diff
@@ -279,10 +287,10 @@ matching element from a collection, you can use ``firstMatch()``::
         'active' => true
     ]);
 
-As you can see from the above, both ``match()`` and ``firstMatch()`` allow you
-to provide multiple conditions to match on. In addition, the conditions can be
-for different paths, allowing you to express complex conditions to match
-against.
+Comme vous pouvez le voir ci-dessus, les deux méthodes ``match()`` et
+``firstMatch()`` vous permettent de fournir plusieurs conditions qui
+correspondent. En plus, les conditions peuvent être pour différents chemins,
+vous permettant d'exprimer des conditions complexes à faire correspondre.
 
```

Comment:
I'll go with this for the entire paragraph :
_Comme vous pouvez le voir ci-dessus, les méthodes match() et firstMatch() vous permettent de fournir plusieurs conditions à matcher. De plus, les conditions peuvent être utilisées sur des chemins différents, vous permettant d'exprimer des conditions complexes à faire correspondre._

I'm open to discussions on this one, I'm not fully convinced of what I have at the moment.